CONTRIBUTING.md

```diff
@@ -60,7 +60,7 @@ See the [README.md](./README.md) for a description of the directory structure to
 
```

Comment:
@josefree should we actually have the directory structure detail inside of here instead of the readme.md directly?